src/ring_theory/local_properties.lean

```diff
@@ -0,0 +1,125 @@
+/-
+Copyright (c) 2021 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import ring_theory.localization
+import data.equiv.transfer_instance
+import group_theory.submonoid.pointwise
+import ring_theory.nilpotent
+
+/-!
+# Local properties of commutative rings
+
+In this file, we provide the proofs of various local properties.
+
+## Naming Conventions
+
+* `localization_P` : `P` holds for `S⁻¹R` if `P` holds for `R`.
```

Comment:
I think it makes sense to try to do the same thing here as in the other PR: turn these conventions in to code. What does it mean for a property `P` to be a "local" property of a ring, and how do these various notions imply each other abstractly.

Also, it seems that you now have two different PRs that both create the same new file. So that will give a conflict at some point. Do you have a preference which one should be merged first and which one should conflict?

Reply:
Either is fine for me, but I would say that this one could go first if the choice had to be made.